src/formatDate.js

```diff
@@ -51,6 +51,84 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  let year;
+  let shortYear;
+  let longYear;
+  let month;
+  let isShortYear = true;
+  let dayOfMonth;
```

Comment:
with coma it would be mistake of lint:

For var declarations, write each declaration in its own statement.
// ✓ ok
var silent = true;
var verbose = true;

// ✗ avoid
var silent = true, verbose = true;

https://mate-academy.github.io/style-guides/javascript-standard-modified